articles/10-tips-for-working-with-node.js/index.md

```diff
@@ -0,0 +1,368 @@
+Alternatively, use `watchers` in `expressions` and `variables`. To start
+watching an `expression`, include `watch ('my expression');` the command
+`watcher` will print active watchers. To remove a watcher you can use line
+`unwatch ('my expression');` Remember to check
+out [MORE](https://www.section.io/engineering-education/debug-nodejs-vscode/)
+ON SECTION.IO efficient ware to deal with bugs from your app.
+
+### 7.  Always keep your code simple and light
+
+It is important to keep the Node.js codebase compact and shallow to
+reduce the latency and speed up the response time of a Node.js
+application. Some devices are slower and their latencies are higher thus
+you need to keep the code small, well-arranged executions and light. The
+same ideas should be applied to your server code as well. While writing
+your Node.js code ensure proper and appropriate naming of things for
+better referencing. This will serve as Documentation of your application
+and make it easier to go back to your code when changes are required.
+Better code quality helps in easier code maintainability
+
+### 8.  Security practices to your server
+
+As you get started with Node.js, try to secure your application by
+proper coding, tooling, and operations.
+
+The main practice to ensure secure Node.js application include
+
+-   Set up the secure Http headers
+
+-   Do a static review of your codebase
+
+-   Handle error carefully (error codes and stack traces)
+
+-   Regularly use strict mode ie restricted variant, undeletable
+    properties, validation, and authentication.
+
+-   Do proper session management such as Http Only
+
+-   Set cookie scope and cookie flags
+
+-   Avoid command injection
+
+-   Check for any vulnerabilities using `retire.js`. Retire.js will assist
+    you to recognize module versions with associated vulnerabilities,
+    install it using command `npm install -g retire`, later after running
+    it, retire command will scan for each vulnerability in your
+    application.
+
+-   Constantly audit all the modules you have in your Node.js.
+    Application using CLI Node Security Platform. Node Security CLI uses
+    Command `nsp`, which allows auditing of `package.json` or
+    `npm-shrinkwrap.json` to check unsafe modules. Run `npm install nsp -global` 
+    from your application project directory `nsp check` to start the check.
+
+Subsequently, practice testing before application deployment to make
+sure that your app passes all security demands.
+
+### 9.  Deploying your Node.js. application
+
+After finishing developing your app and all features are tested, you
+need now be wondering how you will get your application out for the
+world to interact with it. The most familiar Node.js deployment ways if
+either PaaS provider (Heroku) or
+using [Docker](https://nodejs.org/de/docs/guides/nodejs-docker-webapp/).
+
+Let\'s look in brief what you need to deploy to Heroku;
+
+This will happen after the app development process is over and it meets
```

Comment:
You mention at the start of the section "After finishing developing your app" so this is a bit obsolete.